tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Other than the files/paths in "files" and "include", this is almost an exact copy of [`tsconfig.base.json`](https://github.com/Sage-Bionetworks/synapse-web-monorepo/blob/main/shared/tsconfig.base.json) in synapse-react-client. I'm not very familiar with TypeScript compiler options, so if there are more appropriate options, happy to adjust this as needed.